src/frontend/src/components/SearchBar/SearchBar.jsx

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
+import { withApollo, useQuery, useLazyQuery } from 'react-apollo';
+import gql from 'graphql-tag';
 import Paper from '@material-ui/core/Paper';
 import InputBase from '@material-ui/core/InputBase';
 import IconButton from '@material-ui/core/IconButton';
 import SearchIcon from '@material-ui/icons/Search';
 import FormControl from '@material-ui/core/FormControl';
 import NativeSelect from '@material-ui/core/NativeSelect';
 
+const FEED_SEARCH_QUERY = gql`
```

Comment:
You're querying for posts, so naming this FEED_ seems confusing.

Reply:
Sorry this is real rough as I'm taking whatever from docs and playing around with it!